vspace/src/main/java/edu/asu/diging/vspace/web/publicview/ExhibitionModuleController.java

```diff
@@ -31,7 +32,7 @@ public String module(@PathVariable("id") String id, @PathVariable("spaceId") Str
         }
         IModule module = moduleManager.getModule(id);
         model.addAttribute("module", module);
-        if (module == null) {
+        if (module == null || module.getModuleStatus() == null || module.getModuleStatus() == ModuleStatus.UNPUBLISHED) {
```

Comment:
modules without module status should be assumed to be published otherwise if we deploy this all existing modules won't show up anymore.

Reply:
Removed getModuleStatus from null check, treats it as the same as module being published